.github/workflows/spdx.yml

```diff
@@ -0,0 +1,42 @@
+name: Update license data
+
+on:
+  schedule:
+    - cron: '*/60 * * * *'
+
+jobs:
+  spdx:
+    runs-on: ubuntu-latest
+    env:
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Configure Git
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: BrewTestBot
+      
+      - name: Setup Homebrew
+        run: |
+          HOMEBREW_REPOSITORY="$(brew --repo)"
+          rm -rf "$HOMEBREW_REPOSITORY"
+          ln -s "$GITHUB_WORKSPACE" "$HOMEBREW_REPOSITORY"
+      
+      - name: Update license data
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd "$GITHUB_WORKSPACE/Library/Homebrew"
```

Comment:
```suggestion
          cd "$(brew --repo)"
```

Reply:
You mean `"$(brew --repo)/Library/Homebrew"`?

Reply:
Yea, sorry. Disregard this comment.